packages/cli/src/services/dynamicNodeParameters.service.ts

```diff
@@ -182,8 +183,9 @@ export class DynamicNodeParametersService {
 	) {
 		const method = nodeType.methods?.[type]?.[methodName];
 		if (typeof method !== 'function') {
-			throw new Error(
-				`The node-type "${nodeType.description.name}" does not have the method "${methodName}" defined!`,
+			throw new ApplicationError(
+				`Node type "${nodeType.description.name}" does not have the method "${methodName}" defined!`,
```

Comment:
maybe we should have nodeType and methodName only in tags, and keep the message string constant.

Reply:
> keep the message string constant

Thank you - definitely been aiming for this but missed this spot. Kept `nodeType` and `credentialType` for `tags` and all the rest for `extra`.